contrib_bots/lib/github.py

```diff
@@ -0,0 +1,22 @@
+# The purpose of this file is to handle all requests
+# to the Github API, which will make sure that all
+# requests are to the same account, and that all requests
+# authenticated correctly and securely
+# The sole purpose of this is to authenticate, and it will
+# return the requests session that is properly authenticated
+import requests
+import six.moves.configparser, os
+
+def auth():
+    # Sets up the configParser to read the .conf file
+    config = six.moves.configparser.ConfigParser()
+    # Reads the config file
+    config.read([os.path.expanduser('~/.zuliprc-local')])
```

Comment:
Yes, in the contrib-bot [Readme](https://github.com/zulip/zulip/blob/master/contrib_bots/lib/readme.md#third-party-configuration) it says to create a separate config file:
> 63 Do **not** put third party configuration information in your
 64 `.zuliprc` file.  Do not put third party configuration
 65 information anywhere in your Zulip directory.  Instead,
 66 create a separate configuration file for the third party's
 67 configuration in your home directory.